pages/common/lp.md

```diff
@@ -0,0 +1,31 @@
+# lp
+
+> print files
+
+- List printers present on the machine.
+
+`lpstat -p | awk '{print $2}`
+
+- List the default printer.
+
+`lpstat -d`
+
+- Print the output of a command.
+
+`echo "test" | lp -d {{printer_name}}`
```

Comment:
`-d` is demonstrated below, so I'd suggest leaving this example as the simpler `echo "test" | lp` (which also applies the "progressive complexity" principle)